### PR TITLE
scraper/mapped: Add feetToCm post process.

### PR DESCRIPTION
**Pull request by woodgen**
This patch adds a toMetric post process that converts imperial feet and
inches to centimeters.

This implements feature request https://github.com/stashapp/stash/issues/587

For testing try my Jules Jordan performer scraper here:
https://github.com/woodgen/CommunityScrapers/tree/ea%2Bjj

### CODE REVIEW

File: pkg/scraper/mapped.go [hunk 1]
```diff
@@ -276,11 +278,33 @@ func (p *postProcessMap) Apply(value string, q mappedQuery) string {
...
+	reg := regexp.MustCompile("[0-9]+")
+	filtered := reg.FindAllString(value, -1)
+
+	var parsed []float64
+	for _, v := range filtered {
+		k, _ := strconv.ParseFloat(v, 64)
+		parsed = append(parsed, k)
+	}
+
+	var centimeters = parsed[0]*foot_in_cm + parsed[1]*inch_in_cm
```
Comment (WithoutPants) [inline pkg/scraper/mapped.go:296]: This is going to panic if the input only has a single number or doesn't have numbers at all.

```suggestion
	var feet float64
	var inches float64
	if len(filtered) > 0 {
		feet, _ = strconv.ParseFloat(filtered[0], 64)
	}
	if len(filtered) > 1 {
		inches, _ = strconv.ParseFloat(filtered[1], 64)
	}

	centimeters := feet*foot_in_cm + inches*inch_in_cm
```

File: pkg/scraper/mapped.go [hunk 2]
```diff
@@ -276,11 +278,33 @@ func (p *postProcessMap) Apply(value string, q mappedQuery) string {
...
+	// Return rounded integer string
+	return strconv.Itoa(int(math.Round(centimeters)))
+}
+
 type mappedPostProcessAction struct {
 	ParseDate  string                   `yaml:"parseDate"`
 	Replace    mappedRegexConfigs       `yaml:"replace"`
 	SubScraper *mappedScraperAttrConfig `yaml:"subScraper"`
 	Map        map[string]string        `yaml:"map"`
+	ToMetric   bool                     `yaml:"toMetric"`
```
Comment (WithoutPants) [inline pkg/scraper/mapped.go:307]: I think `toMetric` is too generic for this purpose. `feetToCm` would probably be more suitable.

Comment (woodgen) [reply pkg/scraper/mapped.go:307]: This was my first idea as well, but then I noticed it's feet and inches to cm :D Anyway, I will take your suggestion as it's more intuitive.

File: pkg/scraper/mapped.go (hunk 1 above)
Comment (woodgen) [reply pkg/scraper/mapped.go:296]: Thanks for the patch, this is indeed more safe.

File: pkg/scraper/mapped.go [hunk 3]
```diff
@@ -368,6 +395,11 @@ func (a mappedPostProcessAction) ToPostProcessAction() (postProcessAction, error
 		action := postProcessMap(a.Map)
 		ret = &action
 	}
+	if a.FeetToCm {
+		found = "toMetric"
```
Comment (bnkai) [inline pkg/scraper/mapped.go:399]: `found = "toMetric" ` after the refactoring shouldn't it be `feetToCm` ?

Comment (woodgen) [reply pkg/scraper/mapped.go:399]: Oops, missed that one.

**Pull request merged**